frontend/src/consts.ts

```diff
@@ -4,3 +4,27 @@ export const REQUIRED_LEAD_TEC_CREDITS = 6;
 export const ALL_STATUS: Status[] = ['approved', 'pending', 'rejected'];
 export const INITIATIVE_EVENTS = false;
 export const ENABLE_COFFEE_CHAT = true;
+export const ALL_ROLES: Role[] = [
```

Comment:
Keeping the `lead` and `business` roles in for now to maintain backwards compatibility.